docs/CICD_BEST_PRACTICES.md

```
@@ -51,18 +56,21 @@ infra/terraform/
 ```
 
 **State Protection (Implemented)**
+
 - `prevent_destroy = true` on critical resources
 - Separate state per product and environment
 - State locking via Azure Storage
 
 ### Monitoring and Alerts
 
 **Drift Detection (Implemented)**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)